src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_IsByRefLike.cs

```diff
@@ -29,11 +31,26 @@ class Test
 }
 ";
 
-            CompileAndVerify(text, verify: Verification.Passes, symbolValidator: module =>
+
+            void validatePE(PEAssembly assembly)
+            {
+                MetadataReader reader = assembly.GetMetadataReader();
+                var s1 = reader.GetTypeDefinitionHandleByName("S1");
+                Assert.True(assembly.Modules.Single().HasIsByRefLikeAttribute(s1));
```

Comment:
It is up to you if you want to simplify this, but it all can be done much simpler in the ```validate``` below:
```
Assert.True(((PEModuleSymbol)module).Module.HasIsByRefLikeAttribute(((PENamedTypeSymbol)type).Handle));
```